### PR TITLE
ndgGarage ndbit6 v2

### PR DESCRIPTION
**Pull request by ndGarage**
A new version of ndGarage ndbit6 is created. Thanks.

### CODE REVIEW

File: ports/atmel-samd/boards/ndgarage_ndbit6_v2/mpconfigboard.mk [hunk 1]
```diff
@@ -0,0 +1,14 @@
+LD_FILE = boards/samd21x18-bootloader.ld
+USB_VID = 0x239A
+USB_PID = 0x8066
```
Comment (tannewt) [inline ports/atmel-samd/boards/ndgarage_ndbit6_v2/mpconfigboard.mk:3]: Please use a different PID for new boards. (This is the same as the original: https://github.com/adafruit/circuitpython/blob/main/ports/atmel-samd/boards/ndgarage_ndbit6/mpconfigboard.mk#L3 )

**Comment by ndGarage** [comment]
Hi,

I'll request a new PID for this revision and submit a new PR.

I just set up a preliminary page on Hackaday. The link is
https://hackaday.io/project/173537-feathersnow-v2

Thanks,
-Henry

**Comment by ndGarage** [comment]
Have updated my branch with a new USB PID. Not sure if I should create a new PR?

**Comment by dhalbert** [comment]
> Have updated my branch with a new USB PID. Not sure if I should create a new PR?

No new PR is necessary. (A PR is rarely perfect the first time :smiley:) You pushed the change and we will re-review.

**Comment by ndGarage** [comment]
I saw the pre-built failed again due to the CRLF or the end of file things.
I am using a mac. Is it possible to have automatic conversion when I commit
the changes to my branch? I know that .gitattribute can be used for this
purpose. But how?

**Comment by dhalbert** [comment]
I think it's because you added an extra blank line at the end of these files:
Fixing ports/atmel-samd/boards/ndgarage_ndbit6/mpconfigboard.h
Fixing ports/atmel-samd/boards/ndgarage_ndbit6/pins.c

**Pull request merged**